### ISSUE TITLE
Multianimal Pose Tracking: Request for Individualized Colormap for Each Animal

### ISSUE THREAD

**Issue by continig** [DeepLabCut/DeepLabCut#2451]
**Issue**
The current behavior of Deeplabcut's multianimal pose tracking assigns the same color from the colormap to all points of an individual animal. While this provides a clear distinction between animals, it makes it very difficult to distinguish different body parts for a single animal. Furthermore, this does not correspond to the same behavior of the single-animal labelling where the colormap is applied to different points on the skeleton. Below is a screenshot that illustrates the issue:

![image](https://github.com/DeepLabCut/DeepLabCut/assets/144670846/94d851de-fa94-47db-8eec-f5481b8e1880)

**Solution I'd like**
I propose the implementation of a feature that allows users to assign a unique colormap for each individual animal during multianimal pose tracking. This would enhance the visual clarity and ease of analysis when tracking multiple animals in the same video, enabling not only the distinction between different animals but also between different body parts of a single animal.

**Alternatives I've considered**
I simply followed the default config and I observe the behavior described above. Unfortunately the colormap is used for assigning different colors to different animals but the points for each animal is of the same color.

**Additional context**
Config file: [config_current.txt](https://github.com/DeepLabCut/DeepLabCut/files/13424978/config_current.txt)
Deeplabcut version: 2.3.7
OS: Windows 10 Pro

**Comment by MMathisLab** [comment]
@continig for labeling? that (is/was) supported - @jeylau please check some update did not change the behavior ?

For creating videos this already supported!

**Comment by jeylau** [comment]
@MMathisLab, this is slightly different. We do currently have coloring by keypoints or by individuals, but not a mixture of both.
@continig I am not sure this would ease reading, as you would then end up with keypoints with the same name but likely very different colors (or, vice versa, points of very similar color but completely different semantics). I think that toggling the keypoint name display would help, and maybe often switching coloring mode back and forth with the key F. Is that something you've tried or you did not find it useful?

**Comment by MMathisLab** [comment]
>> We do currently have coloring by keypoints or by individuals, but not a mixture of both.

 But I believe she is saying the color by keypoints is not working

(in wx gui we made the edge color the ID)

**Comment by KonradDanielewski** [comment]
Color by individuals or by keypoints is switched with `F` button and works.

We talked with @jeylau today, that maybe a gradient instead of a unique colormap would be something that would help - on the other hand, with more animals and more keypoints the differences become very miniscule if something like a gradient is added. I think a unique colormap may sound good in theory, but in reality would just be very confusing.

**Comment by continig** [comment]
Thank you very much for all your comments. The issue was that the F botton was not working, so I could not switch from a configuration to another one and I continued to visualize the dots as individuals. I honestly cannot say exactly why this was happening, but I solved it but adjusting the settings of the keyboard itself - so outside DLC. 

@KonradDanielewski I perfectly understand the problem of having different colormaps, while having more animals and many bodyparts to label. But I would say that just by switching individuals-keypoints makes just easier to check the labeled frames. 

Thank you very much again for your fast and handy assistance! :)
